From 46993f2b0a69d0ede987d3a67af38ce042a2ed44 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Fri, 31 Jul 2020 08:36:15 -0500
Subject: [PATCH 100/151] crypto: intel_fcs: disable RNG debugging message

commit  f5c17987f2c6a1217622795fc3e25f115b0cf5ba from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Replace dev_info with dev_dbg to disable Random Number Generation (RNG)
debugging message by default.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 drivers/crypto/intel_fcs.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/crypto/intel_fcs.c b/drivers/crypto/intel_fcs.c
index a8786e173751..1338f0f99f66 100644
--- a/drivers/crypto/intel_fcs.c
+++ b/drivers/crypto/intel_fcs.c
@@ -332,9 +332,8 @@ static long fcs_ioctl(struct file *file, unsigned int cmd,
 				return -EFAULT;
 			}
 
-			/* for debug only, will be removed for upstream */
 			for (i = 0; i < 8; i++)
-				dev_info(dev, "output_data[%d]=%d\n", i,
+				dev_dbg(dev, "output_data[%d]=%d\n", i,
 					 *((int *)priv->kbuf + i));
 
 			for (i = 0; i < 8; i++)
-- 
2.26.1

